package.json

```diff
@@ -44,26 +44,26 @@
     "@wdio/sync": "~7.5.2",
     "ajv": "6.12.3",
     "assert": "^2.0.0",
-    "babel-loader": "^8.3.0",
+    "babel-loader": "^8.0.5",
```

Comment:
just an observation but noticed a good few module versions and going back to earlier versions. Is that intentional ?

Reply:
Is this just reverting the previous package.json changes?